Sources/MUXSDKStats/include/MUXSDKStats/MUXSDKPlayerBinding.h

```diff
@@ -39,7 +39,7 @@ typedef NS_ENUM(NSUInteger, MUXSDKViewOrientation) {
 - (void)detachAVPlayer;
 - (void)programChangedForPlayer;
 - (void)prepareForAvQueuePlayerNextItem;
-- (CGRect)getViewBounds;
+- (nullable NSValue *)getViewBounds;
```

Comment:
Ah I see this is technically a public header, hmm. I'm sure no one is relying on it, but let's be safe:
- change the new method to `getViewBoundsValue`
- keep `getViewBounds` but mark it deprecated with `DEPRECATED_ATTRIBUTE`
- implement `getViewBounds` in the base class with something like `[self getViewBoundsValue].CGRectValue` so it technically still works.